panacea/src/pml_parser.yrl

```diff
@@ -127,10 +134,25 @@ action_attribute ->
 
 requires_expr ->
     drug '{' string '}'            : construct('$1', [], value_of('$3')).
-
+requires_expr ->
+    time '{' time_expr_list '}'    : construct('$1', [], value_of_time('$3')).
 requires_expr ->
     expression : '$1'.
 
+time_expr ->
+    years '{' number '}'           : construct('$1', [], value_of('$3')).
```

Comment:
Good point. Floats etc would prob cause the `list_to_integer` call to error as well.